tests/net/utils/src/main.c

```diff
@@ -402,7 +402,7 @@ static bool check_net_addr(struct net_addr_test_data *data)
 	return true;
 }
 
-void test_net_addr(void)
+ZTEST_USER(utils_fn_test_suite, test_net_addr)
```

Comment:
no need for _test_suite here.

Reply:
also, make sure you use net in the testsuite name, to make it networking specific. `utils_fn` can be anything.

Reply:
Updated. Thanks to review